rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -988,3 +989,27 @@ def test_general_cache(globaldb):
             key_parts=[GeneralCacheType.CURVE_POOL_TOKENS, '123'],
         )
         assert values_8 == values_1
+
+
+def test_packaged_db_check_for_constant_assets(globaldb):
+    """Check that if UnknownAsset is raised for asset in CONSTANT_ASSETS, check the packaged db"""
+    # delete one entry in `CONSTANT_ASSETS`
+    with globaldb.conn.write_ctx() as cursor:
+        cursor.execute('DELETE FROM assets WHERE identifier=?;', (A_LUSD.identifier,))
+        assert cursor.rowcount == 1
+    # now resolve the asset and check that no error is raised
+    lusd = A_LUSD.resolve_to_evm_token()
+    assert lusd.asset_type == AssetType.EVM_TOKEN
+    assert lusd.identifier == A_LUSD.identifier
+
+    # change the type of an asset in `CONSTANT_ASSETS`
+    with globaldb.conn.write_ctx() as cursor:
+        cursor.execute('UPDATE assets SET type=? WHERE identifier=?;', (AssetType.FIAT.serialize_for_db(), A_DAI.identifier))  # noqa: E501
+        assert cursor.rowcount == 1
+    # now resolve and check that no error was raised
+    dai = A_DAI.resolve_to_crypto_asset()
+    assert dai.asset_type == AssetType.EVM_TOKEN
+    assert dai.identifier == A_DAI.identifier
```

Comment:
This shouldn't happen. Is not the same problem as if the asset doesn't exists. Here is the user removing it and modifying the globaldb in an unexpected way